modules/olm-catalogsource.adoc

```diff
@@ -2,52 +2,100 @@
-
-There are three primary `sourceTypes` for a `CatalogSource` object:
+A _catalog source_ represents a store of metadata, typically by referencing an _index image_ stored in a container registry. Operator Lifecycle Manager (OLM) queries catalog sources to discover and install Operators and their dependencies. The OperatorHub in the {product-title} web console also displays the Operators provided by catalog sources.
 
-* `grpc` with an `image` reference: OLM pulls the image and runs the pod, which is expected to serve a compliant API.
-* `grpc` with an `address` field: OLM attempts to contact the gRPC API at the given address. This should not be used in most cases.
-* `internal` or `configmap`: OLM parses the ConfigMap data and runs a pod that can serve the gRPC API over it.
+[TIP]
+====
+Cluster administrators can view the full list of Operators provided by an enabled catalog source on a cluster by using the *Administration* -> *Cluster Settings* -> *Configuration* -> *OperatorHub* page in the web console.
+====
 
-The following example defines a catalog source for OperatorHub.io content:
+The `spec` of a `CatalogSource` object indicates how to construct a pod or how to communicate with a service that serves the Operator Registry gRPC API.
 
 .Example `CatalogSource` object
-[source,yaml]
+[%collapsible]
+====
+[source,yaml,subs="attributes+"]
 ----
-apiVersion: operators.coreos.com/v1alpha1
+﻿apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
- name: operatorhubio-catalog
- namespace: olm
+  generation: 1
+  name: example-catalog <.>
+  namespace: {global_ns} <.>
 spec:
- sourceType: grpc
- image: quay.io/operatorhubio/catalog:latest <1>
- priority: -400
- displayName: Community Operators
- publisher: OperatorHub.io
- updateStrategy:
-  registryPoll: <2>
-    interval: 30m
+  displayName: Example Catalog <.>
+  image: quay.io/example-org/example-catalog:v1 <.>
+  priority: -400 <.>
+  publisher: Example Org
+  sourceType: grpc <.>
+  updateStrategy:
+    registryPoll: <.>
+      interval: 30m0s
+status:
+  connectionState:
+    address: example-catalog.{global_ns}.svc:50051
+    lastConnect: 2021-08-26T18:14:31Z
+    lastObservedState: READY <.>
+  latestImageRegistryPoll: 2021-08-26T18:46:25Z <.>
+  registryService: <.>
+    createdAt: 2021-08-26T16:16:37Z
+    port: 50051
+    protocol: grpc
+    serviceName: example-catalog
+    serviceNamespace: {global_ns}
 ----
-<1> Specify catalog image.
-<2> Automatically check for new versions at a given interval to keep up to date.
+<.> Name for the `CatalogSource` object. This name is also used to create the name for the related pod that is created in the requested namespace.
+<.> Namespace to make the catalog available in. To make the catalog available cluster-wide in all namespaces, set this value to `{global_ns}`. The default Red Hat-provided catalog sources also use the `{global_ns}` namespace.
```

Comment:
maybe s/make/create/ the first instance.

because that's the verb used with Kubernetes and for no other reason whatsoever.

(I thought the ISG had guidance to avoid hyphens with company names, but I must have imagined it.)